fr-portfolio.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" type="text/css" href="normalize.css">
+	<link rel="stylesheet" type="text/css" href="portfolio.css">
+	<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+</head>
+<body>	
+	<header class="navbar">
+		<a href="#portfolio">Portfolio</a>
+		<a href="#about">About</a>
+		<a href="#contact">Contact</a>
+	</header>
+	<section class="welcome-banner" >
+							<h1>Freddie's Portfolio</h1>
```

Comment:
Too many tabs / spaces here - also a section containing only 1 element may not be required, you could style the H1 to do the same thing instead (but not sure what is best practice)

Reply:
Originally had an image in this div however Rayna said I should set the image as a background which is attributed to the class welcome-banner but not in the html. So it is ok to have only 1 element in here.